.github/workflows/codeql.yml

```
@@ -0,0 +1,100 @@
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL Advanced"
+
+on:
+  push:
+    branches: [ "testing_gsoc" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)